src/core/namespace.c

```diff
@@ -99,6 +99,7 @@ static const MountEntry protect_kernel_tunables_table[] = {
         { "/sys/kernel/debug",   READONLY,     true  },
         { "/sys/kernel/tracing", READONLY,     true  },
         { "/sys/fs/cgroup",      READWRITE,    false }, /* READONLY is set by ProtectControlGroups= option */
+        { "/sys/fs/selinux",     READWRITE,    true  },
```

Comment:
Do we need writable `/sys/fs/selinux` if `systemd` is compiled with undefined `HAVE_SELINUX`?